### PR TITLE
added proprietary section and new open source resources

### PR DESCRIPTION
**Pull request by rokubun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,16 @@ Have something to add or change? Open a [pull request](https://github.com/barbea
 - **EGNOS Toolkit** ([Download](https://sourceforge.net/projects/libegnos/files/), [Source code](https://sourceforge.net/projects/libegnos/))-  A set of tools to work with Satellite-Based Augmentation Systems (SBAS), specially EGNOS. Support Linux.
 - **GNSS-SDR** ([Download](https://gnss-sdr.org/), [Source code](https://github.com/gnss-sdr/gnss-sdr)) - An open source GNSS software defined receiver. Supports Linux, Mac, and Windows.
 - **RTKLIB** ([Download](http://www.rtklib.com/), [Source code](https://github.com/tomojitakasu/RTKLIB)) - An Open Source Program Package for GNSS Positioning. Has a companion Android app [RTKGPS+](https://play.google.com/store/apps/details?id=gpsplus.rtkgps). Supports Windows.
+- **gAGE Lab** [Download](https://gage.upc.es/gLAB/) A free and publicly available
+software for GNSS data processing.
+
+### Proprietary tools
+
+- [Jason Positioning-as-a-Service](https://jason.rokubun.cat) An online service
```
Comment (barbeau) [inline README.md]: Could you please describe which services are free for users of any GNSS receiver, and which require payment or purchase of a specific receiver?

Comment (rokubun) [reply README.md]: Thanks Sean for your comment, I've pushed a commit commenting on this. I will perform a pull request to accept this change. Best!

**Comment by barbeau** [comment]
Thanks @rokubun! I tweak a few entries and the organization and will merge now. Please keep me posted when the Jason Positioning-as-a-Service website exits beta and is no longer free for all - at that point I'll make a decision about whether it stays on the list.

**Comment by rokubun** [comment]
Hello Sean
We will definitely let you know, but in any case, Jason will have a freemium business model, with a monthly free tier that will allow free use for all the non-intensive users.
Regards,

**Pull request merged**